brands/amenity/bank.json

```
@@ -932,6 +931,15 @@
       "official_name": "Bank of the Philippine Islands"
     }
   },
+  "amenity/bank|BPI Family Savings Bank": {
+    "countryCodes": ["ph"],
+    "tags": {
+      "amenity": "bank",
+      "brand": "BPI Family Savings Bank",
+      "brand:wikidata": "Q94480105",
+      "name": "BPI Family Savings Bank",
```

The only thing that's causing your PR to fail is this comma.

```suggestion
      "name": "BPI Family Savings Bank"
```